CHANGELOG.md

```diff
@@ -12,6 +12,12 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
+* Workaround for AppCode issue with emojis in Build Phases. 
```

Comment:
we shouldn't call anyone out -- simply `Stop using emojis in build phase names, switching to a [CP] prefix.`